test/core/client_channel/lb_policy/BUILD

```diff
@@ -201,7 +210,10 @@ grpc_cc_test(
         "gtest",
     ],
     language = "C++",
-    tags = ["no_test_ios"],
+    tags = [
+        "lb_unit_test",
```

Comment:
Same here: This tag isn't needed, since this is a config parser test, not an LB policy test.

Reply:
done